docs/CONTRIBUTING.md

```diff
@@ -27,6 +27,34 @@ Formatting rules for eclipse can be found [here](../code_style.xml).
 Provided patches should be contained and should not modify code outside of the scope of the patch.
 This will make it quicker to perform reviews and merging the pull requests.
 
+## Logging
+
+Use this as a guideline for what to log for each level, also use the isXyzEnabled (isTraceEnabled, isDebugEnabled etc)
+before logging if the log message that contains anything else than already defined variables.
```

Comment:
I would put the "isEnabled" text below the table, right before the actual example.